card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -4,6 +4,11 @@
   self.drop_item Env.params['drop_item'] if Env.params['drop_item']
 end
 
+event :insert_item_event, :before=>:approve, :on=>:save, :when=> proc {|c| Env.params['insert_item']} do
+  index = Env.params['item_index'] || 0
+  self.insert_item index.to_i, Env.params['insert_item']
+end
+
```

Comment:
Seems like a good idea.  Note the 'self.' on line 9 and similar on line 4 (which didn't change) isn't needed.  At least I don't think they are, and generally I think we should avoid self. whenever possible.  Otherwise I start looking for a reason it is needed.